nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-connection-configuration.js

```diff
@@ -189,7 +189,7 @@ nf.ConnectionConfiguration = (function () {
 
             // populate the port information
             $('#input-port-source').show();
-            $('#input-port-source-name').text(inputPortName);
+            $('#input-port-source-name').text(inputPortName).addClass('ellipsis').attr('title', inputPortName);
```

Comment:
Can the ellipsis class for all fields changed in this file be applied directly to the element in the corresponding JSP?